EFCore.NamingConventions/NamingConventions/Internal/NamingConventionsOptionsExtension.cs

```diff
@@ -11,17 +11,23 @@ public class NamingConventionsOptionsExtension : IDbContextOptionsExtension
     {
         DbContextOptionsExtensionInfo _info;
         NamingConvention _namingConvention;
+        CultureInfo _cultureInfo;
 
-        public NamingConventionsOptionsExtension() {}
 
-        protected NamingConventionsOptionsExtension([NotNull] NamingConventionsOptionsExtension copyFrom)
-            => _namingConvention = copyFrom._namingConvention;
+        public NamingConventionsOptionsExtension(CultureInfo info = null) {}
```

Comment:
Please remove this constructor and the one below - they shouldn't exist (note there weren't any before this PR). The pattern to follow for options extensions is to start with an empty instance, and each time clone it via the appropriate With method.